frontend/public/components/storage/attach-storage.tsx

```diff
@@ -259,31 +249,22 @@ class AttachStorageForm extends React.Component<
               .
             </div>
           )}
+          <label className="control-label co-required" htmlFor="volume-name">
```

Comment:
`htmlFor` looks wrong

Reply:
That was an existing bug.  Should I remove the tag since that label covers multiple items now including radio input, dropdown, and CreatePVCForm?

Reply:
Yeah, I would just take it out